query/query.go

```diff
@@ -2210,6 +2212,10 @@ func ProcessGraph(ctx context.Context, sg, parent *SubGraph, rch chan error) {
 			}
 
 			filter.SrcUIDs = sg.DestUIDs
+			if len(filter.SrcUIDs.Uids) == 0 {
```

Comment:
Is there an existing test for this scenario?

Reply:
I added a test `TestFilterWithNoSrcUid` here.